x-pack/plugins/observability/common/index.ts

```diff
@@ -5,7 +5,10 @@
  * 2.0.
  */
 
-export type { AsDuration, AsPercent } from './utils/formatters';
+export type { AsDuration, AsPercent, TimeUnitChar } from './utils/formatters';
+
+export { formateDurationFromTimeUnitChar } from './utils/formatters';
```

Comment:
Change to `formatDurationFromTimeUnitChar`

Reply:
Done